src/FOHClasses/Order.java

```diff
@@ -0,0 +1,75 @@
+package FOHClasses;
+
+import FOHClasses.Collection.DishCollection;
+
+public class Order {
+    private final int orderInt;
+    private DishCollection items;
+    private String specialRequests;
```

Comment:
Please check the requirements document sent to team 8 for the decided attribute name.